### PR TITLE
add prediction certainty methods

### PR DESCRIPTION
**Pull request by jasperdev**
Hi Joris,

These functions allow for example for the usage of all the ROC functionality in scikit-learn.

### CODE REVIEW

**Comment by MrNuggelz** [comment]
Hey Jasper,
thanks for the contribution. It looks like one of the errors is in line 265 of sklearn_lvq/glvq.py.
`res = dist[:,self.c_w_ != 0].min(1) - dist[:,self.c_w_ == 0].min(1)`
The problem is that this works only for two classes and only when theses classes have label 0 and 1.
The problem with your predict_proba in sklearn_lvq/rslvq.py is that `_costf` can't be used to get the posterior.
I hope this will help you.

**Comment by MrNuggelz** [comment]
Not really sorry, but i am working on a function for that.
